en/flight_modes/takeoff.md

```diff
@@ -28,50 +28,96 @@ Parameter | Description
 <span id="MIS_TAKEOFF_ALT"></span>[MIS_TAKEOFF_ALT](../advanced_config/parameter_reference.md#MIS_TAKEOFF_ALT) | Target altitude during takeoff (default: 2.5m)
 <span id="MPC_TKO_SPEED"></span>[MPC_TKO_SPEED](../advanced_config/parameter_reference.md#MPC_TKO_SPEED) | Speed of ascent (default: 1.5m/s)
 <span id="COM_RC_OVERRIDE"></span>[COM_RC_OVERRIDE](../advanced_config/parameter_reference.md#COM_RC_OVERRIDE) | Controls whether stick movement on a multicopter (or VTOL in MC mode) causes a mode change to [Position mode](../flight_modes/position_mc.md). This can be separately enabled for auto modes and for offboard mode, and is enabled in auto modes by default.
-<span id="COM_RC_STICK_OV"></span>[COM_RC_STICK_OV](../advanced_config/parameter_reference.md#COM_RC_STICK_OV) | The amount of stick movement that causes a transition to [Position mode](../flight_modes/position_mc.md) (if [COM_RC_OVERRIDE](#COM_RC_OVERRIDE) is enabled).
+<span id="COM_RC_STICK_OV"></span>[COM_RC_STICK_OV](../advanced_config/parameter_reference.md#COM_RC_STICK_OV) | The amount of stick movement that causes a transition to [Position mode](../flight_modes/position_mc.md) (if [COM_RC_OVERRIDE](#COM_RC_OVERRIDE) is enabled)
 
 <span id="fixed_wing"></span>
-## Fixed Wing (FW)
+## Fixed-wing (FW)
+
+Automatic takeoff has two modalities: *catapult/hand-launch* or *runway takeoff*.
+The mode defaults to catapult/hand launch, but can be set to runway takeoff by setting [RWTO_TKOFF](#RWTO_TKOFF) to 1.
+
+There are two ways to start an automatic takeoff on fixed-wing vehicles: either by [planning a mission takeoff](../flight_modes/mission.md#takeoff) and starting the mission, or by switching to the _Takeoff mode_ and arming the vehicle.
+
+In both cases, a flight path (starting point and takeoff course) and clearance altitude are defined.
+The flight path takes the vehicle's current position as starting point when the takeoff mode is first entered, and a straight line from this starting point continues in the direction of the defined course indefinitely.
+On takeoff, the aircraft will follow this line, climbing at the maximum climb rate ([FW_T_CLMB_MAX](../advanced_config/parameter_reference.md#FW_T_CLMB_MAX)) until reaching the clearance altitude.

```

Comment:
For a takeoff mode the heading comes from the launch direction. 

For mission mode I'm not certain. What I think happens is that the [TAKEOFF](https://mavlink.io/en/messages/common.html#MAV_CMD_NAV_TAKEOFF) item defines the lat, lon, altitude. 
- Is the altitude in the item the clearance altitude? 
- If you mission launch you'll initially be thrown or head off along runway. That direction might not match the mission item lat/lon. At what point do you head towards that lat/lon?

Reply:
- Yes that's the clearance altitude.
- it will immediately go towards the mission Takeoff WP lat/lon (keep bearing from start of takeoff to Takeoff WP until past clearance altitude).

Reply:
No need to answer ^^^. What I've done here is more clearly separated the mission mode and takeoff mode differences. Just a paragraph reorg.